abc.templates/README.md

```diff
@@ -3,6 +3,7 @@
 ## Installation with abc CLI
 
 1. Render infrastructure template and actuate terraform.
+> A domain is required for GitHub Token Minter.
```

Comment:
Can you expand on this a little bit? NOTE: Before you begin, you should have a domain and access to update DNS information for the publicly facing Cloud Run service to be accessible by GitHub.....